src/adafruit_blinka/microcontroller/generic_linux/libgpiod_pin.py

```diff
@@ -31,10 +31,10 @@ def __init__(self, pin_id):
         self.id = pin_id
         if isinstance(pin_id, tuple):
             self._num = int(pin_id[1])
-            self._chip = gpiod.Chip(str(pin_id[0]), gpiod.Chip.OPEN_BY_NUMBER)
+            self._chip = gpiod.chip(str(pin_id[0]), gpiod.chip.OPEN_BY_NUMBER)
         else:
             self._num = int(pin_id)
-            self._chip = gpiod.Chip("gpiochip0", gpiod.Chip.OPEN_BY_NAME)
+            self._chip = gpiod.chip("gpiochip0", gpiod.chip.OPEN_BY_NAME)
```

Comment:
Same here.

Reply:
```
def open(
        self, device: Union[int, str], how: int = _CHIP_OPEN_LOOKUP
    ) -> None:
        """
        @brief Open a GPIO chip.

        @param device: String or int describing the GPIO chip.
        @param how:    Indicates how the chip should be opened.

        If the object already holds a reference to an open chip, it will be
        closed and the reference reset.

        Usage:
            chip.open("/dev/gpiochip0")
            chip.open(0, chip.OPEN_BY_NUMBER)
        """
        device = str(device)
        func = open_funcs[how]

        chip_struct = func(device)
        if chip_struct is None:
            errno = get_errno()
            raise OSError(
                errno,
                strerror(errno),
                "cannot open GPIO device {}".format(device),
            )

        self._m_chip = shared_chip(chip_struct)
```

Reply:
Ok, I'll have to see if something has changed for some of the other boards, just to make sure it still works.

Reply:
You can say what version you are using and i can try with her :)